### PR TITLE
fix: update tree, query cluster info and send telemetry in backgroud (#912)

### LINKED ISSUE #912: When launching the plugin on an existing project, I cannot use it. IndicatorCancellationException
**Steps:**
1. ASSERT: use a project that is already indexed.
1. ASSERT: have RH Sandbox as current cluster
1. EXEC: launch the plugin

**Result:**
The resource tree shows an error and thus most actions are not available in the context menus:

ps. I am still not fully sure how to/not to reproduce it. It happens always to me.

Workaround for me: I touch `~/.kube/config` which causes the resource tree to be refreshed and display the cluster, current namespace etc.

### PR DESCRIPTION
**Pull request by adietish**
fixes #912

### CODE REVIEW

File: src/main/java/org/jboss/tools/intellij/openshift/utils/odo/OdoCli.java [hunk 1]
```diff
@@ -418,7 +415,7 @@ public ComponentInfo getComponentInfo(String project, String component, String p
 
   private ComponentInfo parseComponentInfo(String json, ComponentKind kind) throws IOException {
     JSonParser parser = new JSonParser(Serialization.json().readTree(json));
-    return parser.parseDescribeComponentInfo(kind, isPodmanPresent);
+    return parser.parseDescribeComponentInfo(kind, isPodmanPresent());
```
Comment (sbouchet) [inline src/main/java/org/jboss/tools/intellij/openshift/utils/odo/OdoCli.java:418]: why modify this variable to a method call each time a compoenent is parsed ? this is time consuming for no reason, has podman is present or not for the whole time the extension is instanciated.

Comment (adietish) [reply src/main/java/org/jboss/tools/intellij/openshift/utils/odo/OdoCli.java:418]: the reason is the following: the check was executed in the constructor and it's a heavy operation. All the heavy operations that are in the constructor of OdoCli get executed when the tree structure requests the odo binary. This then lead to the tree displaying "IndicatorCancelled" when the UI was waiting for too long for these operations to be finished.
In this particular case, the instance variable isn't even needed. There's only 1 place where it's value is needed. That's where I replaced the instance var by a direct call.

**Review by sbouchet — APPROVED** [review]
LGTM. heavy check is now perfomed only once, in a lazy manner. good one !

**Pull request merged**